sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

```diff
@@ -2,15 +2,15 @@
 
 Azure Key Vault is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, or certificates. A vault is a logical group of secrets.
 
-The Azure Key Vault administration library clients supports administrative tasks such as full backup / restore and key level role based access control (RBAC).
+The Azure Key Vault administration library clients support administrative tasks such as full backup / restore and key-level role-based access control (RBAC).
 
 ## Getting started
 
 ### Install the package
 Install the Azure Key Vault administration client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-Install-Package Azure.Security.KeyVault.Administration
+Install-Package Azure.Security.KeyVault.Administration --version 4.1.0-preview.1
```

Comment:
This will need to change to 4.2.0-preview.1 at some point. May want to do it now.